sintr_common/lib/tasks.dart

```diff
@@ -253,7 +285,7 @@ class TaskController {
 
     // TODO: Add co-ordination of the job to outside the control scripts
     var query = _db.query(_TaskModel)
-      ..filter("lifecycleState =", READY_STATE);
+      ..filter("lifecycleState =", READY_STATE)..limit(10);
```

Comment:
Looking at this in hindsight, it is all a bit too magical. I'll factor it out slightly and pull this out into configuration. This is the window of items that the worker tries to allocate to itself before giving up (returning null, and going to sleep). Typically the first item will be taken. The window size is a tradeoff between datastore reads and probably of waste associated with the longish sleep delay if all n items are hit by the race.